iotedgedev/envvars.py

```diff
@@ -209,6 +209,26 @@ def save_envvar(self, key, value):
             self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
             sys.exit(-1)
 
+    def get_registries(self):
+        registries = {}
+        self.CONTAINER_REGISTRY = {}
+        for key in os.environ:
+            key = key.lower()
+            if key.startswith('container_registry_server'):
+                token = key[len('container_registry_server'):]
+                if token not in registries:
+                    registries[token] = {}
+                registries[token]['server'] = os.environ[key]
+            elif key.startswith(('container_registry_username', 'container_registry_password')):
+                token = key[len('container_registry_username'):]
```

Comment:
Consider defining a var like (sorry about the long name) KEY_LENGTH_FOR_USERNAME_OR_PASSWORD. This would create some self-describing code for future readers/maintainers. Right now, they might have a brief moment of doubt before tying together that username and password are the same length.